source/linux/Foundational_Components/Tools/Flash_via_Fastboot.rst

```diff
@@ -0,0 +1,126 @@
+.. include:: /_replacevars.rst
```

Comment:
Update this line based on this PR
https://github.com/TexasInstruments/processor-sdk-doc/pull/85